test/k8sT/manifests/iperf3-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: iperf3-server
+  labels:
+    id: iperf3-server
+    zgroup: testapp
+spec:
+  hostNetwork: true
+  terminationGracePeriodSeconds: 0
+  containers:
+    - name: iperf
+      image: docker.io/cilium/netperf:2.0
```

Comment:
Could you also update other deployments using that image in a separate commit? :pray:

Reply:
Of course!

Reply:
I think you forgot this ;-)

Reply:
Oh! Sorry! I understand that you are saying in another PR. I will do that.

Reply:
Done!